src/component/Router.js

```diff
@@ -30,7 +23,7 @@ function RouterComponent() {
                     <JoinList />
                 </Route>
                 <Route path="">
-                    { checkToken === null ? <Home/> : <FetchItems/> }
+                    <Home />
```

Comment:
Non-blocking - we still need the conditional routing, so I'm not sure why the ternary was deleted? But this'll get changed anyway when we lift state so I assume we don't need to worry about it :)

Reply:
Forgot to mention, yes, I was thinking that we won't need it once we finish "week2cleanup"